sys/dev/cy/cy_pci.c

```diff
@@ -94,21 +92,24 @@ static driver_t cy_pci_driver = {
 };
 
 DRIVER_MODULE(cy, pci, cy_pci_driver, cy_devclass, 0, 0);
-MODULE_PNP_INFO("U32:device", pci, cy, cy_devs,
-    sizeof(cy_devs[0]), nitems(cy_devs) - 1);
+PCI_PNP_INFO(cy_devs);
 MODULE_DEPEND(cy, pci, 1, 1, 1);
 
 static int
 cy_pci_probe(dev)
 	device_t dev;
 {
+	const struct pci_device_table *cyd;
 	u_int32_t device_id;
 
 	device_id = pci_get_devid(dev);
 	device_id &= ~0x00060000;
```

Comment:
This mask is troublesome. It means that we're looking at 4x the number of cards...

Reply:
Should we try and get rid of this mask then ?